openedx/core/lib/cache_utils.py

```diff
@@ -173,7 +174,7 @@ def __init__(self, namespace=None, model=None, cache_time=86400):
         if model:
             post_save.connect(self.invalidate, sender=model)
             post_delete.connect(self.invalidate, sender=model)
-            namespace = str(model)
+            namespace = "{}.{}".format(model.__module__, model.__qualname__)
```

Comment:
This was the best I could figure out. Thanks to IRC folks for the qualname tip!

Reply:
Is there a reason we don't take the string representation of the model and just replace the spaces with underscores?  Trying to understand why we are using these dunder values instead of doing that.  Is it because there are other ways in which the model representation can be invalid?

Reply:
Rather than a model instance, this parameter takes a model class. This was producing cache keys like `":1:<class 'xblock_django.models.XBlockConfiguration'>.xblock_django.api.deprecated_xblocks"` and the new version would instead produce `":1:xblock_django.models.XBlockConfiguration.xblock_django.api.deprecated_xblocks"`.